daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -1320,9 +1324,30 @@ func startCatchupTest(t *testing.T, catchpoint string, nodeError error, expected
 	req := httptest.NewRequest(http.MethodPost, "/", nil)
 	rec := httptest.NewRecorder()
 	c := e.NewContext(req, rec)
-	err := handler.StartCatchup(c, catchpoint)
+	var err error
+	if initRounds != 0 {
+		err = handler.StartCatchup(c, catchpoint, model.StartCatchupParams{Initialize: &initRounds})
+	} else {
+		err = handler.StartCatchup(c, catchpoint, model.StartCatchupParams{})
+	}
```

Comment:
With the parameter change, this all becomes.
```suggestion
	err := handler.StartCatchup(c, catchpoint, model.StartCatchupParams{Initialize: &initRounds})
```

Reply:
I left this in so that the `nil` parameter handling would still be exercised